test/module.test.js

```diff
@@ -320,6 +330,113 @@ describe('basic', () => {
   })
 })
 
+describe('hreflang', () => {
+  let nuxt
+
+  beforeAll(async () => {
+    const override = {
```

Comment:
This might in the end give same result but merging these locales with base config ends up with duplicate locales:

```
[ { code: 'en', iso: 'en', name: 'English' },                                                                                                                               
         { code: 'fr', iso: 'fr-FR', name: 'Français' },
         { code: 'en', iso: 'en', name: 'English' },
         { code: 'fr', iso: 'fr-FR', name: 'Français' },
         { code: 'es', iso: 'es-ES', name: 'Spanish (Spain)' },
         { code: 'esVe',
           iso: 'es-VE',
           name: 'Spanish (Venezuela)',
           isCatchallLocale: true } ]
```

because deepmerge will merge array of locales. I'd prefer to not have duplicates in the first place by just loading original config and then overriding locales manually. Like in `lazy loading` testsuite for example.

Reply:
Totally makes sense! Thanks for the heads up ☺️